### ISSUE TITLE
404 on cljstyle

### ISSUE THREAD

**Issue by licht1stein** [DeLaGuardo/setup-clojure#102]
Hi, I think cljstyle changed the naming of their files:

```
"https://github.com/greglook/cljstyle/releases/download/0.16.626/cljstyle_0.16.626_linux.zip". Code(404) Message(Not Found)
```

**Comment by licht1stein** [comment]
They added amd64 postfix:

![image](https://github.com/DeLaGuardo/setup-clojure/assets/35418634/fc27a094-292c-46ac-b3cf-86cac006999c)

**Comment by DeLaGuardo** [comment]
this is fixed in 12.3 version. please update your workflow definition

**Comment by licht1stein** [comment]
Thanks!
